packages/shared-base/src/Settings/index.ts

```diff
@@ -0,0 +1,37 @@
+import { defer } from '@dimensiondev/kit'
```

Comment:
Please do not add new Settings API based on ValueRef.

Use `packages/shared-base/src/kv-storage`. You can find an example at `packages/mask/shared/kv-storage.ts`